### ISSUE TITLE
Task2: Logistic Regression

### ISSUE THREAD

**Issue by Zhou198** [Zhou198/Algorithm-Combing#2]
## 6. Dealing with imbalanced sample
* **Change weights:** We can change the weights for different classes in loss function when training. Say, improve weight for the class with less sample size and decrease that for class with more sample size.
* **Add extra sample:** Add more data for the class whose sample size is less. Obviously, this method will cost much money and time.
* **Oversampling:** Sample on the small class with replacement until the sample size is close to the large one.
* **Undersampling:** Take a sub-sample on the large class, such that the sample size close to the small one. Actually, this method wastes a part of resource.

## 7. Parameters of Sklearn for Logistic Regression
